commons-core/src/test/scala/com/avsystem/commons/serialization/JGenCodecTest.scala

```diff
@@ -0,0 +1,39 @@
+package com.avsystem.commons
+package serialization
+
+import java.lang.annotation.RetentionPolicy
+
+import com.github.ghik.silencer.silent
+
+import scala.collection.immutable.ListMap
+
+@silent
+class JGenCodecTest extends JCodecTestBase {
```

Comment:
Why can't this be also moved to shared sources?

Reply:
Missing implementations in Scala.js. I'll move working parts "java collection test" to cross-compiled tests.